enterprise/backend/src/metabase_enterprise/sandbox/api/util.clj

```diff
@@ -59,9 +59,10 @@
   []
   (boolean
    (when-not *is-superuser?*
-     (if *current-user-id*
-       (seq (enforced-sandboxes-for *current-user-id*))
-       ;; If no *current-user-id* is bound we can't check for sandboxes, so we should throw in this case to avoid
-       ;; returning `false` for users who should actually be sandboxes.
-       (throw (ex-info (str (tru "No current user found"))
-                       {:status-code 403}))))))
+    (if *current-user-id*
+      (let [enforced-sandboxes (enforced-sandboxes-for *current-user-id*)]
```

Comment:
Is this functionally different than before?

Reply:
nope — I think I changed this while trying to fix a different issue and didn't revert it. I'll switch it back.